docs/documentation/helpcenter.md

```diff
@@ -315,104 +284,238 @@ Zulip browser window in a screenshot; instead, it should only show
+  information and warning blocks.
+
+- Instructions can be formatted with [Tab](#tabs),
+  [Steps/FlattenSteps](#steps-and-flattenedsteps) and
+  [NavigationSteps](#navigationsteps) components.
 
 ### Icons
 
-See [icons documentation](../subsystems/icons.md). Icons should always be named
-with their in-app tooltip or brief action name, _not_ the name of the icon in
-the code.
+See [icons documentation](../subsystems/icons.md). Icons should always be
+referred to with their in-app tooltip or a brief action name, _not_ the
+name of the icon in the code.
 
-### Macros
+When using these icons in an MDX file, they act as any other component:
 
-**Macros** are elements in the format of `{!macro.md!}` that insert common
-phrases and steps at the location of the macros. Macros help eliminate
-repeated content in our documentation.
+```
+import SquarePlusIcon from "~icons/zulip-icon/square-plus"
 
-The source for macros is the Markdown files under `help/include` in the
-[main Zulip server repository](https://github.com/zulip/zulip).
+Click the **new topic** (<SquarePlusIcon />) button next to the name of the channel.
+```
 
-- **Administrator only feature** `{!admin-only.md!}`: Notes that the feature
-  is only available to organization administrators.
+For the import statement, the icon component should be named as the camel
+case of the icon name, with any dashes removed, followed by `Icon`, e.g.,
+in the example above `square-plus` is imported as `SquarePlusIcon`.
 
-- **Message actions** `{!message-actions.md!}`: First step to navigating to
-  the on-hover message actions.
+### Include files
 
-- **Message actions menu** `{!message-actions-menu.md!}`: Navigate to the
-  message actions menu.
+You can include any file inside another MDX file as a regular import, which
+helps to eliminate repeated content in our documentation.
 
-- **Save changes** `{!save-changes.md!}`: Save changes after modifying
-  organization settings.
+All our include files live at `starlight_help/src/content/docs/include`,
+and can be imported and used as a regular component:
 
-- **Channel actions** `{!channel-actions.md!}`: Navigate to the channel actions
-  menu from the left sidebar.
+```
+import AdminOnly from "./include/_AdminOnly.mdx";
 
-- **Start composing** `{!start-composing.md!}`: Open the compose box.
+<AdminOnly />
+```
+
+If you're adding a new include file, make sure to have an underscore at the
+beginning of the file name as that ensures the file won't be considered a
+standalone article in the help center.
+
+A lot of our include files are list macros, i.e., they are partial lists
+that are part of bigger lists. When the partial list is an ordered list,
+it needs to be wrapped in a [FlattenedSteps](#steps-and-flattenedsteps)
+component.
+
+Note that if the include file has any h2 or h3 headers (`##`, `###`), then
+you will need to add the same headers to any files where you are importing
+and using that include file:
```

Comment:
I think we should also say that if these headers can be avoided in the include file, they should be avoided. We should also mention the reason to mention these h2 or h3 headers as described in #35127 and https://chat.zulip.org/#narrow/channel/19-documentation/topic/new.20help.20center.3A.20on.20this.20page.20overview